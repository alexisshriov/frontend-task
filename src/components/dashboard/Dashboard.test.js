import React from 'react';
import { shallow } from 'enzyme';
import Dasboard from './Dashboard';

describe('MyComponent', () => {
  it('should render Dasboard correctly', () => {
    const component = shallow(<Dasboard />);
    expect(component).toMatchSnapshot();
  });
});