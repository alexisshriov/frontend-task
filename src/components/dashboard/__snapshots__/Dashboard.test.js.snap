// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render Dasboard correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
